lib/ohai/plugins/kernel.rb

```diff
@@ -116,7 +116,9 @@ def os_lookup(sys_type)
     so = shell_out("env lsmod")
     so.stdout.lines do |line|
       if line =~ /([a-zA-Z0-9\_]+)\s+(\d+)\s+(\d+)/
```

Comment:
I personally like putting a comment with the line I'm matching against above a regex like this, so without leaving the code you can figure out what $1 is supposed to be.